packages/utils/src/useTableTools/FilterConfigBuilder/ChipBuilder.js

```diff
@@ -0,0 +1,52 @@
+import { conditionalFilterType } from '@redhat-cloud-services/frontend-components/ConditionalFilter';
```

Comment:
All of these "FilterConfigBuilder" classes are moved over from the inventory-compliance package, from where it is used in all tables in Compliance with filters: https://github.com/RedHatInsights/frontend-components/tree/master/packages/inventory-compliance/src/Utilities